<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhPreferNumbers</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_CadClie">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Procurar cliente por nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Nome">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome do cliente</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nome">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonProcurar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(103, 146, 255);</string>
    </property>
    <property name="text">
     <string>Procurar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 177, 177);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 47, 50);</string>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cadastro">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>460</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 188, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>label_CadClie</zorder>
   <zorder>label_Nome</zorder>
   <zorder>lineEdit_nome</zorder>
   <zorder>pushButtonProcurar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imagem.qrc"/>
 </resources>
 <connections/>
</ui>
